src/content/docs/accounts/accounts-billing/account-structure/new-relic-account-structure.mdx

```diff
@@ -26,7 +26,7 @@ This section is about organizations on the New Relic One account/user model, not
 
 At New Relic, an "organization" represents a New Relic customer. The organization contains everything relevant to a New Relic customer: its accounts, its users, and its data. 
 
-A New Relic "account" can be considered a workspace. For example, you might have an account for a specific app, or a set of related hosts and services for a specific initiative or project, or you might have an account for a specific team.
+A New Relic "account" can be considered a workspace. For example, you might have an account for a specific app, or a set of related hosts and services for a specific initiative or project, or you might have an account for a specific team. Each account has its own [account ID](/docs/accounts/accounts-billing/account-setup/account-id), which is relevant when doing some procedures, like making API calls. 
```

Comment:
how about simplifying to "Each account has its own account ID. They're used to make API calls and other procedures."